etc/zabbix/zabbix_agentd.d/ic_azure.conf

```diff
@@ -22,10 +22,13 @@ UserParameter=azure.discover.metrics.namespace[*],source /opt/digiaiiris/virtual
 
 # Azure dimension discovery. Parameters: configuration file, resource, metric, dimension
 UserParameter=azure.discover.dimensions[*],source /opt/digiaiiris/virtualenv/zabbix-azure-monitoring/bin/activate && azure_discover_dimensions "$1" "$2" "$3" "$4"
+UserParameter=azure.discover.multidimensions[*],source /opt/digiaiiris/virtualenv/zabbix-azure-monitoring/bin/activate && azure_discover_dimensions "$1" "$2" "$3" "$4" --extent "$5"
```

Comment:
Please add a separate comment for the new UserParameter items, stating their parameters